### ISSUE TITLE
Fix for the no-empty rule constructors with readonly parameters 

### ISSUE THREAD

**Issue by cyrilgandon** [palantir/tslint#1629]
Following this issue https://github.com/palantir/tslint/pull/68/files fixed by this PR https://github.com/palantir/tslint/pull/68, and with the new possibility with TS2.0 to use the `readonly` declaration, the rule is no more complete.

### Bug Report

- __TSLint version__: `3.15.1`
- __TypeScript version__: `2.0`

#### TypeScript code being linted

```ts
class Foo {
    static $inject = ['$cookies'];
    constructor(readonly $cookies: angular.cookies.ICookiesService) {
    }
}
```
#### Actual behavior

`block is empty`

#### Expected behavior

no tslint error
